docs/multicast-guide.md

```diff
@@ -0,0 +1,153 @@
+  - [Inbound and outbound multicast traffic statistics](#inbound-and-outbound-multicast-traffic-statistics)
+  - [Multicast NetworkPolicy statistics](#multicast-networkpolicy-statistics)
+- [Use case example](#use-case-example)
+- [Limitations](#limitations)
+  - [Traffic in local network control block](#traffic-in-local-network-control-block)
+  - [Linux kernel](#linux-kernel)
+  - [Encap mode](#encap-mode)
+<!-- /toc -->
+
+## Prerequisites
+
+`Multicast` is introduced in v1.5 as an alpha feature. The feature gate
+`Multicast` must be enabled in the `antrea-controller` and `antrea-agent`
+configuration.
+
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable multicast traffic.
+      Multicast: true
+  antrea-agent.conf: |
+    # Enable multicast traffic.
+      Multicast: true
+    multicast:
+    # The names of the interfaces on Nodes that are used to forward multicast traffic.
+    # Defaults to transport interface if not set.
+      multicastInterfaces: 
+
+    # The interval at which the antrea-agent sends IGMP queries to Pods.
+    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+      igmpQueryInterval: "125s"
+```
+
+## Multicast NetworkPolicy
+
+Antrea NetworkPolicy and Antrea ClusterNetworkPolicy are supported for the following
+types of multicast traffic:
+
+1. IGMP egress rules: IGMP membership report and IGMP leave group messages.
+2. IGMP ingress rules: IGMP query, which includes IGMPv1, IGMPv2, and IGMPv3.
+3. Multicast egress rules: non-IGMP multicast traffic from Pod to other Pods or external hosts.
+Note: multicast ingress rules are currently not supported.
+
+Examples: [ACNP for IGMP traffic](antrea-network-policy.md#acnp-for-igmp-traffic) and
+[ACNP for multicast egress traffic](antrea-network-policy.md#acnp-for-multicast-egress-traffic).
+Please also check the meanings of special fields for Multicast NetworkPolicies from the paragraphs
+below these examples.
+
+## Debugging and collecting multicast statistics
+
+Users could check following guides and corresponding tools to get the current status of
+the multicast application running in the cluster.
+
+### Pod multicast group information
+
+This command prints the cluster-wide information of Pods that has joined multicast group
+throughout the whole cluster. Example:
+
+```bash
+kubectl get multicastgroups
+GROUP       PODS
+225.1.2.3   default/mcjoin, namespace/pod
+224.5.6.4   default/mcjoin
+```
+
+### Inbound and outbound multicast traffic statistics
+
+Please refer to the corresponding [antctl page](antctl.md#multicast-commands).
+
+### Multicast NetworkPolicy statistics
+
+[Antrea NetworkPolicystats](feature-gates.md#networkpolicystats) also apply to multicast NetworkPolicy.
+
+## Use case example
+
+In this section, we will show how multicast video steaming works in
+Antrea using [VLC](https://www.videolan.org/vlc/).
+
+To start a sample video streaming server, we create a Pod streaming sample video
```

Comment:
To start a video streaming server

Reply:
we start a VLC Pod to stream a sample video

Reply:
updated